src/components/parameters/short-circuit/short-circuit-study-area-tab-panel.tsx

```
@@ -91,6 +93,37 @@ export const ShortCircuitStudyAreaTabPanel = forwardRef<HTMLSpanElement, Readonl
             />
         );
 
+        useEffect(() => {
+            const checkSelectedFilters = async () => {
+                const genericFilters: { id: UUID; name: string, deleted?: boolean }[] = getValues(
+                    `${SPECIFIC_PARAMETERS}.${NODE_CLUSTER_FILTER_IDS}`
+                );
+                genericFilters.forEach(async (filter) => {
+                    filter.deleted = false;
```

ESLint

[error] Assignment to property of function parameter 'filter'.

(no-param-reassign)